tests/example.file.php

```diff
@@ -72,7 +72,7 @@ public function getPrice()
     {
         const BASE_PRICE = 1;
 
-        function getPrice();
+        public function getPrice();
```

Comment:
think this is unintended. There is a testcase that checks the default accessor of a method.

Reply:
This applies to all changes in this file.